enhancements/microshift/audit-log-configuration-options.md

```diff
@@ -0,0 +1,255 @@
+| WriteRequestBodies             | In addition to logging metadata for all requests, logs request bodies for every write request to the API servers (create, update, patch, delete, deletecollection). This profile has more resource overhead than the Default profile. |
+| AllRequestBodies               | In addition to logging metadata for all requests, logs request bodies for every read and write request to the API servers (get, list, create, update, patch). This profile has the most resource overhead.                            |
+
+MicroShift will reuse the existing OpenShift custom resource, `apiserver.config.openshift.io`, defined as part of the [OpenShift API](https://github.com/openshift/api/blob/master/config/v1/types_apiserver.go) as part of the translation process from "profile" to "policy". Logic for translating the OpenShift profiles to the `policy.audit.k8s.io` objects is provided by the [openshift/library-go](https://github.com/openshift/library-go/tree/release-4.14/pkg/operator/apiserver/audit) library and is intended for import into external code bases.
+
+To enable OpenShift profiles, MicroShift will get the `apiserver.auditLog.profile` value from the MicroShift config and internally wrap this value into an `APIServer.config.openshift.io/v1` API object. This instance will be passed to the [library-go GetAuditPolicy function](https://github.com/openshift/library-go/blob/release-4.14/pkg/operator/apiserver/audit/audit_policies.go#L93), which will return the `policy.audit.k8s.io/v1` equivalent.
+
+**Included Dependencies**
+
+- **openshift/library-go**: already imported by MicroShift.
+- **openshift/api**: will be added to MicroShift's direct dependency list.
+
+### Risks and Mitigations
+
+- Exceeding disk capacity: Microshift targets small form-factor devices with limit on-board storage. If the product of `maxFileSize` and `maxFiles` equals a size larger than the available storage, the apiserver risks destabilizing the system.  This can be mitigated via documentation which recommends the user understand their storage limitations when setting these values.
+- Lost Log Data: The apiserver culls log files given a certain size or age. If users do not take care to back up logs at a rate greater than the rate at which the kube-apiserver culls the files, they risk losing log data. This can be mitigated with examples of log-forwarding provided in documentation, example manifests, or both. Alternatively, it may be necessary to consider deploying the openshift-cluster-logging-operator to provide supportable log forwarding features.
+- Exposing the apiserver's audit-log-path, which allows users to set a custom log location, would hinder sos report gathering. Instead, users should replace `/var/log/kube-apiserver` with a symlink the desired path.  Sos will follow the symlink and collect logs by default.
+
+### Drawbacks
+
+- N/A
+
+## Open Questions
```

Comment:
Both of these questions are answered. Since both result in no changes, they could be moved to the Alternatives section.

Reply:
1 question was already covered in the alts, so only moving the 2nd of the 2.